my-app/src/shared/components/VideoPlayer.js

```diff
@@ -0,0 +1,11 @@
+import { observer } from 'mobx-react';
+
+export const VideoPlayer = observer((videos) => {
```

Comment:
please don't use iframe since it has some major security vulnerabilities https://blog.bitsrc.io/4-security-concerns-with-iframes-every-web-developer-should-know-24c73e6a33e4#:~:text=By%20default%2C%20content%20from%20an,user%20into%20a%20phishing%20website.&text=In%20the%20above%20code%2C%20there,site%20embedded%20using%20an%20iframe.

instead consider using following package https://www.npmjs.com/package/react-player, it can work with embeded youtube videos directly